srcpkgs/river/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'river'
 pkgname=river
-version=0.2.3
+version=0.2.4
 revision=1
 build_style=zig-build
-configure_args="$(vopt_if xwayland -Dxwayland) -Dpie"
+configure_args="$(vopt_if xwayland -Dxwayland) -Dpie -Dstrip"
```

Comment:
-Dstrip will break the river-dbg package

Reply:
done